discord/application.go

```diff
@@ -6,24 +6,26 @@ import (
 )
 
 type Application struct {
-	ID                  Snowflake        `json:"id"`
-	Name                string           `json:"name"`
-	Icon                *string          `json:"icon,omitempty"`
-	Description         string           `json:"description"`
-	RPCOrigins          []string         `json:"rpc_origins"`
-	BotPublic           bool             `json:"bot_public"`
-	BotRequireCodeGrant bool             `json:"bot_require_code_grant"`
-	TermsOfServiceURL   *string          `json:"terms_of_service_url,omitempty"`
-	PrivacyPolicyURL    *string          `json:"privacy_policy_url,omitempty"`
-	Owner               *User            `json:"owner,omitempty"`
-	Summary             string           `json:"summary"`
-	VerifyKey           string           `json:"verify_key"`
-	Team                *Team            `json:"team,omitempty"`
-	GuildID             *Snowflake       `json:"guild_id,omitempty"`
-	PrimarySkuID        *Snowflake       `json:"primary_sku_id,omitempty"`
-	Slug                *string          `json:"slug,omitempty"`
-	CoverImage          *string          `json:"cover_image,omitempty"`
-	Flags               ApplicationFlags `json:"flags,omitempty"`
+	ID                  Snowflake           `json:"id"`
+	Name                string              `json:"name"`
+	Icon                *string             `json:"icon,omitempty"`
+	Description         string              `json:"description"`
+	RPCOrigins          []string            `json:"rpc_origins"`
+	BotPublic           bool                `json:"bot_public"`
+	BotRequireCodeGrant bool                `json:"bot_require_code_grant"`
+	TermsOfServiceURL   *string             `json:"terms_of_service_url,omitempty"`
+	PrivacyPolicyURL    *string             `json:"privacy_policy_url,omitempty"`
+	InstallationParams  *InstallationParams `json:"install_params"`
+	Tags                *[]string           `json:"tags"`
```

Comment:
```suggestion
	Tags                []string            `json:"tags"`
```
this should not be a pointer

Reply:
the `tags` key is missing if there are no tags, it doesn't return an empty array, so it'd have to be nil, wouldn't it?

Reply:
Slices can be nil already so it's fine